velox/docs/functions/spark/array.rst

```diff
@@ -15,6 +15,15 @@ Array Functions
 
         SELECT array(1, 2, 3); -- [1,2,3]
 
+.. spark:function:: array_append(array(E), element) -> array(E)
+
+    Add the ``element`` at the end of the ``array`` passed as first argument.
+    Type of element should be same to type of the elements of the ``array``.
```

Comment:
nit: same -> the same
type of the elements of the array -> the type of elements in the array

Reply:
Done